### DISCUSSION TITLE
Update Naked Card

### DISCUSSION THREAD

**Discussion by BiancaMach** [Telefonica/mistica-design#1676] [Proposals]
### Goal

Avaliar a possibilidade de inserir um padding-right no bloco de componentes de texto no Naked Card para ter maior área de respiro no carrossel.

### Context

Atualmente, quando há mais de um card, o espaçamento utilizado não dá respiro suficiente para os blocos de texto, sendo assim necessário inserir um padding-right para que essa área de respiro seja maior.

### Impact

Em situações onde há grande quantidade de texto, a área de respiro maior deixará a leitura mais acessível.

### Discussion Questions

Como está atualmente:
<img width="730" alt="Screenshot 2024-02-23 at 16 38 57" src="https://github.com/Telefonica/mistica-design/assets/153837409/dbec78e9-3cbd-4a70-84cf-51bcc719a667">

<img width="735" alt="Screenshot 2024-02-23 at 16 39 12" src="https://github.com/Telefonica/mistica-design/assets/153837409/db9ba9eb-1889-402a-8d2d-62af71d7406c">

### Do you have a visual proposal?

https://www.figma.com/file/tmIfzrqKmIUellqazmapgs/Update-Naked-Card?type=design&node-id=331%3A5823&mode=design&t=KzAdyypowLLAASrp-1

**Comment by yceballost** [comment]
Hi Bianca!

it is a good point! We think this change makes a lot of sense to avoid the content getting mixed up with the cards next to it

I have a doubt... this applies also to `Small Naked Card`? Or just for `Naked Card`?

It is 16px also for Small Naked Card?

**Reply by aweell** [reply]
Another question: should apply only in mobile? Or all screen sizes?

https://tinyurl.com/2cgllwzo

I thinks that works fine in desktop even if the gap increases to 16px

**Reply by BiancaMach** [reply]
Yes! Applies to both cards and all screen sizes using 16px too .

**Comment by aweell** [comment]
Hi @BiancaMach we're going to create an issue to update the naked specs & component with your proposal. Thank you for your contribution.

**Comment by yceballost** [comment]
@BiancaMach in order to follow the process the best as possible, open a ticket in OBVIVO jira board, please! :)

**Reply by BiancaMach** [reply]
Ok! here it is https://jira.tid.es/browse/OBVIVO-2417
